src/components/Burger/BuildControls/BuildControls.js

```diff
@@ -1,36 +1,42 @@
-import React from "react";
-import classes from "./BuildControls.css";
-import BuildControl from "./BuildControl/BuildControl";
+import React from 'react';
+import classes from './BuildControls.css';
+import { BuildControl } from './BuildControl/BuildControl';
 
 const controls = [
-  { label: "Salad", type: "salad" },
-  { label: "Bacon", type: "bacon" },
-  { label: "Cheese", type: "cheese" },
-  { label: "Meat", type: "meat" }
+  { label: 'Salad', type: 'salad' },
+  { label: 'Bacon', type: 'bacon' },
+  { label: 'Cheese', type: 'cheese' },
+  { label: 'Meat', type: 'meat' }
 ];
 
-const buildControls = props => (
-  <div className={classes.BuildControls}>
-    <p>
-      Current Price: <strong>{props.price.toFixed(2)}</strong>
-    </p>
-    {controls.map(ctrl => (
-      <BuildControl
-        key={ctrl.label}
-        label={ctrl.label}
-        added={() => props.ingredientAdded(ctrl.type)}
-        removed={() => props.ingredientRemoved(ctrl.type)}
-        disabled={props.disabled[ctrl.type]}
-      />
-    ))}
-    <button
-      className={classes.OrderButton}
-      disabled={!props.purchaseable}
-      onClick={props.ordered}
-    >
-      ORDER NOW
-    </button>
-  </div>
-);
-
-export default buildControls;
+export function BuildControls({
```

Comment:
I already like this way more because at least you're explicitly aware of what parameters you can use rather than taking a guess at what's inside `props`. 👍

Reply:
Yeah, even in general and not specific to react, object destructuring is one of those things that can delay the need to adopt things like typescript etc